frb_example/with_flutter/rust/src/api.rs

```diff
@@ -103,3 +107,28 @@ pub fn off_topic_deliberately_panic() -> i32 {
     std::env::set_var("RUST_BACKTRACE", "1"); // optional, just to see more info...
     panic!("deliberately panic!")
 }
+
+pub struct SumWith {
+    pub x: u32,
+}
+
+impl SumWith {
+    pub fn sum(&self, y: u32) -> u32 {
```

Comment:
uber-nit: would be great to mention that, we deliberately copy SumWith and other code into here because we want to test the flag